asynq/debug.pyi

```diff
@@ -9,6 +9,9 @@ from ._debug import options as options
 def DUMP_ALL(self, value: None = ...) -> bool: ...
 @overload
 def DUMP_ALL(self, value: bool) -> None: ...
+def enable_original_exc_handler(enable: bool) -> None: ...
```

Comment:
This API feels weird for disabling. i.e. async.debug. `enable_original_exc_handler(enable=False)`

Reply:
Yeah I guess, but I expect this only to be called with enable=True normally since it's default off.

Do you remember why were calling the original exc handler in the first place?

Reply:
I think pylons and pyramid rely on these exception hooks to make those web based debuggers work

Reply:
Hmm, pylons seems to be fine, but it did break some ipython debugger functions. Unfortunately not possible keep nicer printing behavior and make those work. People using that functionality can turn the original exception handlers on.